modules/etcd-node-scaling.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-performance-scale-practices/recommended-etcd-practices.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="etcd-node-scaling_{context}"]
+= Node scaling for etcd
+
+In general, clusters must have 3 control-plane nodes. However, if your cluster is installed on a bare metal platform, you can scale a cluster to up to 5 nodes as a post-installation task.
+
+For example, to scale from 3 to 4 nodes post installation, you add a host and install it as a control-plane node. Then, the etcd Operator scales accordingly to account for the additional node.
+
+Scaling to 4-node or 5-node clusters is available only on bare metal platforms for clusters that were installed by using the Assisted Installer.
```

Comment:
Not limited to Assisted Installer only, must be bare metal, resulting cluster from assisted installer, agent-based installer, UPI or IPI will be the same on day-2

Reply:
Good catch. I'll update that.